### ISSUE TITLE
Retro Achievements prevents some games from loading

### ISSUE THREAD

**Issue by lasers** [libretro/RetroArch#3866]
Retro Achievements prevents me from playing a Nintendo game "Mario is Missing" on Nestopia core... I think this game does not have Retro Achievement list or something else therefore CHEEVOS won't start anything without one... whatever that is.

The process is still stuck running and not showing anything on the monitor.
```
...
RetroArch [libretro INFO] :: [Nestopia]: Machine is NTSC.
RetroArch [INFO] :: CHEEVOS testing NES (discards VROM).
```

**Comment by twinaphex** [comment]
@leiradel Any thoughts?

**Comment by leiradel** [comment]
Yes, that was a bogus commit, the issue was fixed in PR #3862. Sorry about that.

**Comment by lasers** [comment]
I'm trying to test this, but I'm facing odd issue with extracting the game.

RetroArch [ERROR] :: Failed to extract content from compressed file: /home... blah.
RetroArch tells me I have 7zip + zlib extraction support.

I don't know if it's something on my side because I'm messing around too much... or if it's coming from git... Verison: 5a70656

**Comment by leiradel** [comment]
Hm, this is not related to cheevos. Can you test with an uncompressed ROM?

**Comment by lasers** [comment]
Yes. I'm only trying to do the same thing I did last time and that is to load a same game from the history content. I'll load the game manually. I think loading the game differently could lead to different results.

The uncompressed game loads fine with all cheevos options enabled. 😄  Maybe the problem I'm now having is related to history menu... so I tested that... and it's not. The same thing occurred when I'm loading the game from NES game list. Weird.

It's all good now? Have you see my other RetroArchivement issue? About rewinding?
